bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/RuleEngineImpl.java

```diff
@@ -103,6 +104,14 @@
 @NonNullByDefault
 public class RuleEngineImpl implements RuleManager, RegistryChangeListener<ModuleType> {
 
+    @ComponentPropertyType
```

Comment:
This pattern is new in OSGi Compendium 7 too? Am I right?

Underscores will be replaced by "." thus the property name still is "rule.reinitialization.delay"?

Reply:
It should :wink: 
You could have a look at the spec or to get a very short overview, have a look at: https://www.eclipsecon.org/europe2017/sites/default/files/slides/Constructor%20injection%20and%20other%20new%20features%20for%20Declarative%20Services%201.4.pdf page 14ff

Reply:
Yes, thanks. I found and read that presentation too.